src/open_inwoner/haalcentraal/utils.py

```diff
@@ -41,4 +39,4 @@ def update_brp_data_in_db(user, initial=True):
     if initial is False:
         user.save()
 
-    system_action(_("data was retrieved from haal centraal"), content_object=user)
+    logger.info("Retrieving data for %s from haal centraal based on BSN", user)
```

Comment:
Another one logger that was a timelinelog.

Reply:
See previous comment